docs/content/050-ingest-example.adoc

```diff
@@ -0,0 +1,37 @@
+[[ingest-example]]
+=== Ingest Example
+
+GeoWave can ingest any data type that has been listed as an ingest plugin. Let's start out with the GeoTools datastore; this
+wraps a bunch of GeoTools supported formats. This includes all file-based datastores supported within GeoTools.
+We will use the shapefile capability for our example here.
+
+==== Something recognizable
+
+The naturalearthdata side has a few shapefile we can use use. On the page
+http://www.naturalearthdata.com/downloads/50m-cultural-vectors/[50m Cultural Vectors]
+
+Let's download the Admin 0 - Countries shapefile:
+http://naciscdn.org/naturalearth/50m/cultural/ne_50m_admin_0_countries.zip[ne_50m_admin_0_countries.zip]
+
+[source, bash]
+----
+$ mkdir ingest
+$ mv ne_50m_admin_0_countries.zip ingest/
+$ cd ingest
+$ unzip ne_50m_admin_0_countries.zip
+$ rm ne_50m_admin_0_countries.zip
+$ cd ..
+$ geowave -localingest \
```

Comment:
It is. I opened issue #737 to address the examples. There are several others that have to be updated as well.